.travis.yml

```diff
@@ -37,20 +37,26 @@ matrix:
       language: minimal
       install:
         - docker-compose build
+        - pip install codecov --user
       script:
         - docker-compose run backend ./runtests.sh
+      after_success:
+        - codecov -f coverage.xml -t 995256be-3a1f-4d10-b181-e24f88603b6b
 
     - env: python-tests-main
       language: minimal
       install:
         - docker-compose build
+        - pip install codecov --user
```

Comment:
@armenzg are we able to run coverage locally?  I thought my suggestion of `/tests/coverage.sh` was good as an example, and to ensure coverage could be run locally.

Reply:
I just saw this comment. I have a new branch to get this out the door so we can postpone anything that is missing to a follow up issue.

Support for running code coverage results locally is supported as part of this PR.

Now, submitting results from localhost to **codecov** requires setting up a personal codecov account, grabbing a token, installing `codecov` manually and then running the tests.